ledger/src/blocktree.rs

```diff
@@ -1073,19 +1073,22 @@ impl Blocktree {
         shred_start_index: u64,
         _max_entries: Option<u64>,
     ) -> Result<Vec<Entry>> {
-        self.get_slot_entries_with_shred_count(slot, shred_start_index)
+        self.get_slot_entries_with_shred_info(slot, shred_start_index)
             .map(|x| x.0)
     }
 
-    pub fn get_slot_entries_with_shred_count(
+    /// Returns the entry vector for the slot starting with `shred_start_index`, the number of
+    /// shreds that comprise the entry vector, and whether the last shred has been received for
```

Comment:
We are returning `is_full()`, so the actual condition is stronger than `last shred has been received`, it's every shred up to the shred marked to the last shred has been received